### PR TITLE
Make node-webcrypto work for me

### PR DESCRIPTION
**Pull request by hildjj**
Not sure where you got `.default` from, but it doesn't seem to be in the version of node-webcrypto I just downloaded.  Ignore this PR if you've already fixed.

I'm about to checkin a transition to the ava test framework if you want to pause for a moment.  I think you'll like what that does for your tests.

### CODE REVIEW

**Comment by hildjj** [comment]
There's the testing switchover.  Ready to merge, assuming I didn't misunderstand the `.default` thing.

**Comment by hildjj** [comment]
Make sure to do an `npm install`, then do `npm run coveragehtml` and open `coverage/index.html` in a browser.

File: test.js [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 let cbor = require('cbor');
 
-let node_webcrypto_ossl = require('node-webcrypto-ossl').default;
+let node_webcrypto_ossl = require('node-webcrypto-ossl');
+console.log(node_webcrypto_ossl)
```
Comment (erdtman) [inline test.js]: maybe remove this line

Comment (hildjj) [inline test.js]: Removed the whole file. :)  sorry I left that in the commit though.

**Pull request merged**